### PR TITLE
Fix "unknown type uint8_t" error

### PR DESCRIPTION
**Pull request by Craft4Cube**
Added missing include to "inttypes.h"

### CODE REVIEW

**Comment by rogerclarkmelbourne** [comment]
Can you list the steps to reproduce the error

**Comment by GitMoDu** [comment]
Why not just #include <stdint.h> where needed?

**Comment by ArminJo** [comment]
@rogerclarkmelbourne Please merge it fast,
You can see errors [here](https://github.com/ukw100/IRMP/runs/1395315243?check_suite_focus=true#step:3:659)
and [here](https://github.com/ArminJo/ServoEasing/runs/1396135525?check_suite_focus=true#step:3:903)
and more [here](https://github.com/ukw100/IRMP/actions/runs/361418390)
**It crashes all my CI Workflows using your core!**

**Comment by ArminJo** [comment]
And please consider to release it fast, since all my compiles crashes.

**Comment by stevstrong** [comment]
@ArminJo ,
the best solution is to change in line 34
```
uint8_t negative = 0;
```
to 
```
unsigned char negative = 0;
```
Please change this in you PR and then I will commit it.

File: STM32F1/cores/maple/avr/dtostrf.c [hunk 1]
```diff
@@ -21,6 +21,8 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
+#include <inttypes.h>
```
Comment (stevstrong) [inline STM32F1/cores/maple/avr/dtostrf.c:24]: Remove this line.

**Comment by ArminJo** [comment]
@stevstrong I totally agree, but the PR is not from me.
But of course I can provide a new PR for this, if it helps, but maybe it is easier to just change the line manually and then close this PR?
And thanks for the fast reply! 👍

**Comment by stevstrong** [comment]
This should be fixed by https://github.com/rogerclarkmelbourne/Arduino_STM32/commit/194a45ae256c6ede786d58525fe1b7550c9e6614.

**Comment by ArminJo** [comment]
T H A N K S 👍

**Pull request closed without merging**